incubator/consul/templates/consul-petset.yaml

```diff
@@ -157,6 +157,12 @@ spec:
       - name: gossip-key
         secret:
           secretName: gossip-key
+      readinessProbe:
+        httpGet:
+          path: /v1/health/service/consul
```

Comment:
yah we have had a problem with that endpoint, when using DNS names for discovery.